components/fxa-client/src/device.rs

```diff
@@ -239,11 +239,10 @@ pub struct Device {
 /// executing these commands are encapsulated as part of the FxA Client component,
 /// so consumers simply need to select which ones they want to support, and can
 /// use the variants of this enum to do so.
-///
-/// In practice, the only currently-supported command is the ability to receive a tab.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
 pub enum DeviceCapability {
     SendTab,
+    CloseRemoteTabs,
```

Comment:
🚲🎨🏡 time! Do we want to call this capability `CloseRemoteTabs`? `CloseTabs`? `CloseTab`? `CloseSynced{Tabs, Tab}`?

It doesn't affect the implementation, but might help to have a catchy, consistent name for our feature.

Reply:
I like `CloseTabs` (which is what I used at first! :joy:) or `CloseTab`, mirroring @mhammond's [comment](https://phabricator.services.mozilla.com/D204556?id=836796#inline-1137804) on the Desktop patch—it sidesteps the question of "which is the remote device and which is the local device".

Reply:
I hate naming :) In this case I think `CloseTabs` is correct for exactly the reason you say - but even then it's slightly ambigious - does the capability say "I can have my tabs closed" or "I can close other device's tab"? Regardless, I think `CloseTabs` is perfectly reasonable and better than the other options.

Reply:
I like `CloseTab(s)` as well! Keeps it consistent with SendTab!

Reply:
`CloseTabs` it is! 😁